### DISCUSSION TITLE
Build Benchmarks

### DISCUSSION THREAD

**Discussion by TornaxO7** [eneoli/flakeshot#66] [General]
Ok, so after adding cranelift + mold, here are the differences:

# Now (with cranelift + mold)
Tested on commit `b87f271`.

| Command | Mean [s] | Min [s] | Max [s] | Relative |
|:---|---:|---:|---:|---:|
| `cargo build` | 8.498 ± 0.195 | 8.283 | 8.661 | 1.00 |

# Vanilla (just simply stable rust)
Tested on commit `96b1842`.

| Command | Mean [s] | Min [s] | Max [s] | Relative |
|:---|---:|---:|---:|---:|
| `cargo build` | 9.600 ± 0.196 | 9.421 | 9.809 | 1.00 |

**Comment by eneoli** [comment]
What's the issue here @TornaxO7 ?

**Comment by TornaxO7** [comment]
> What's the issue here @TornaxO7 ?

It's just an issue to discuss about the build performance of `flakeshot`. Just a general thing. If we're talking about discussions: Why not enabling them?

**Comment by eneoli** [comment]
Don’t we use mold and carnelift already?

> If we're talking about discussions: Why not enabling them?

sure ^^

**Reply by TornaxO7** [reply]
> Don’t we use mold and carnelift already?

We do! But there will be some [upcoming improvements](https://blog.rust-lang.org/2023/11/09/parallel-rustc.html) and I thought it would be cool to see it's impact on the project :)
